tests/unit/type/stream-cgroups.tcl

```diff
@@ -1395,14 +1395,16 @@ start_server {
         test {XREADGROUP from PEL inside MULTI} {
             # This scenario used to cause propagation of EXEC without MULTI in 6.2
             $replica config set propagation-error-behavior panic
+            $master hello 3
             $master del mystream
             $master xadd mystream 1-0 a b c d e f
             $master xgroup create mystream mygroup 0
             $master xreadgroup group mygroup ryan count 1 streams mystream >
             $master multi
             $master xreadgroup group mygroup ryan count 1 streams mystream 0
             set reply [$master exec]
-            assert_equal $reply {{{mystream {{1-0 {a b c d e f}}}}}}
+            assert_equal $reply {{mystream {{1-0 {a b c d e f}}}}}
```

Comment:
@guybe7 yes, but the test failed under RESP3, so I changed the test to always run under RESP3.
please note that i removed a `{}`.

Reply:
the reason why i don't fix it under RESP2 is `hello 2` doesn't work with flag `--force-resp3`.